pa-types.md

```diff
@@ -0,0 +1,71 @@
+## Public administration types
+
+The key `meta/pa-type` defines the types of public administration which is
+expected to use this software. This document contains a list of allowed
+values for that key. While the key is optional, if it is specified it
+must contain a value from this list.
+
+### International
+
+Value | Description
+----- | -----------
+`city` | City
+`hospital` | Hospital
+`police` | Police forces
+`school` | School of any level
+`university` | University of any level
+
+### Italy
```

Comment:
Awesome, perhaps we can think of some method of extending the base set non-centrally.